pages/common/github-label-sync.md

```diff
@@ -0,0 +1,24 @@
+# github-label-sync
+
+> A command line interface for synchronising GitHub labels.
+> Homepage: <https://npmjs.com/package/github-label-sync>.
+
+- Synchronise labels using a local "labels.json" file:
+
+`github-label-sync --access-token {{token}} {{repository_name}}`
+
+- Synchronise labels using a specific labels JSON file:
+
+`github-label-sync --access-token {{token}} --labels {{url|path/to/json_file}} {{repository_name}}`
+
+- Synchonise labels with a dry run:
+
+`github-label-sync --access-token {{token}} --dry-run {{repository_name}}`
+
+- Keep labels that aren't in "labels.json":
```

Comment:
Perhaps this will read better?
```suggestion
- Keep labels that aren't in `labels.json`:
```

Is this right? I can't remember whether filenames should be in backticks in descriptions or not

Reply:
I've applied the other changes, I haven't yet applied this as I never know which they're meant to use.